test_complete.sh

```
@@ -0,0 +1,549 @@
+    cd examples/simple
+
+    run terraform init; [ "$status" -eq 0 ]
+    run terraform validate; [ "$status" -eq 0 ]
+    run terraform plan -out=simple.tfplan; [ "$status" -eq 1 ]
+}
+
+@test "color codes are properly formatted in output functions" {
+    source "$ORIGINAL_PWD/test_complete.sh"
+    [ "$GREEN" = $'\033[0;32m' ]
+    [ "$RED"   = $'\033[0;31m' ]
+    [ "$YELLOW"= $'\033[1;33m' ]
```

Shellcheck (0.10.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)